projects-dashboard/src/content/project/cashapp_hermit.md

```diff
@@ -0,0 +1,30 @@
+---
+repo:
+  owner: cashapp
+  name: hermit
```

Comment:
When you see the CI badge with no status it's likely because the default branch is different than main.
<img width="421" alt="image" src="https://github.com/user-attachments/assets/b5e30d92-6d7a-45ea-af8b-38c8c8fcdc6f" />

To adjust that, just set this config: https://github.com/block/ospo/blob/main/projects-dashboard/src/content/config.ts#L6

For this project it should be `mainBranch: master` (which I just realized its a bad config name, it should be renamed to `defaultBranch`.